src/include/solver/pca.h

```diff
@@ -12,7 +12,7 @@ typedef struct params {
   int X_n;
   int X_m;
   int k;
-  const char *algorithm;
+  char *algorithm;
```

Comment:
I think we can change this to `char` with no issues. Also, tests seem to be passing.